### PR TITLE
Add private hosting + polish

### PR DESCRIPTION
**Pull request by akegaviar**
Added private hosting changes.
Did some polishing of the existing docs.

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -54,6 +55,7 @@ module.exports = {
                     '/platform/delete-a-network',
                     '/platform/delete-a-project',
                     '/platform/manage-your-organization-identity',
```
Comment (easeev) [inline docs/.vuepress/config.js:57]: Why not identities?

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -54,6 +55,7 @@ module.exports = {
...
+                    '/platform/manage-your-organization-integration',
```
Comment (easeev) [inline docs/.vuepress/config.js:58]: Why not organizations?

File: docs/platform/manage-your-organization-integration.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+meta:
+  - name: description
+    content: Add and delete your organization's private cloud integrations
+  - name: keywords
+    content: private amazon aws cluster integration blockchain deploy kubernetes k8s
+---
+
+# Manage your organization's integration
+
+You can add [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/) or a self-managed Kubernetes cluster as an integration to deploy your nodes and networks to it.
```
Comment (easeev) [inline docs/platform/manage-your-organization-integration.md:11]: If we mention options that are not available yet I suggest to include other options in clouds as well (GKE, AKS)

File: docs/platform/supported-hosting-options.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    content: Deploy your blockchain nodes and networks to your own Kubernetes cluster.
+  - name: keywords
+    content: amazon eks blockchain hosting cluster kubernetes k8s
+---
+
+# Supported hosting options
+
+Chainstack supports the following hosting options:
+
+* Chainstack-provided — deployment to cloud providers. See [Supported cloud providers](/platform/supported-cloud-hosting-providers).
```
Comment (easeev) [inline docs/platform/supported-hosting-options.md:13]: I'd rephrase somehow to explain that this option is built-in in Chainstack and nodes are being deployed to the infrastructure that we manage/maintain/etc

File: docs/platform/supported-hosting-options.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+* Private — deployment to a Kubernetes cluster. See [Manage your organization's integration](/platform/manage-your-organization-integration).
```
Comment (easeev) [inline docs/platform/supported-hosting-options.md:14]: Here I'd explain that this option is about deploying into the infrastructure that you own (without mentioning Kubernetes here)

File: docs/.vuepress/config.js (hunk 1 above)
Comment (akegaviar) [reply docs/.vuepress/config.js:57]: Done

File: docs/.vuepress/config.js (hunk 2 above)
Comment (akegaviar) [reply docs/.vuepress/config.js:58]: Done

File: docs/platform/manage-your-organization-integration.md (hunk 3 above)
Comment (akegaviar) [reply docs/platform/manage-your-organization-integration.md:11]: Done

File: docs/platform/supported-hosting-options.md (hunk 4 above)
Comment (akegaviar) [reply docs/platform/supported-hosting-options.md:13]: Done

File: docs/platform/supported-hosting-options.md (hunk 5 above)
Comment (akegaviar) [reply docs/platform/supported-hosting-options.md:14]: Done

File: docs/platform/manage-your-organizations-integrations.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+## Add Amazon EKS as an integration
+
+1. On Chainstack, click <a href="https://console.chainstack.com/integrations" target="_blank">Integrations</a>.
+1. Provide any name.
+1. Under **Type**, select **Amazon Elastic Kubernetes Service**.
+1. Select the region where you have a deployed Amazon EKS service.
+1. Provide the access key and secret of the user you have specifically created for the Chainstack integration.
+1. Provide the Kubernetes namespace you have specifically created for the Chainstack integration.
+1. Provide any domain name for the deployment. This domain name will be a part of your node's endpoint URL.
```
Comment (neewy) [inline docs/platform/manage-your-organizations-integrations.md:30]:
```suggestion
1. Provide a domain name. This domain name will be used as a root domain name, and the nodes you deploy will be accessible via generated subdomain endpoint URL. 
```

File: docs/platform/manage-your-organizations-integrations.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+1. Click **Add connection**.
+
+Once the integration is validated, you can use it to deploy your nodes and networks.
```
Comment (neewy) [inline docs/platform/manage-your-organizations-integrations.md:33]: Should we tell how long this takes on average?

Comment (neewy) [inline docs/platform/manage-your-organizations-integrations.md:33]:
```suggestion
Once the integration is validated, which takes 5 minutes on average, you can use it to deploy your nodes and networks.
```

File: docs/platform/manage-your-organizations-integrations.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+## Delete an integration
+
+1. On Chainstack, click <a href="https://console.chainstack.com/integrations" target="_blank">Integrations</a>.
+1. Select an integration.
+1. Click **Edit** > **Delete**.
```
Comment (neewy) [inline docs/platform/manage-your-organizations-integrations.md:39]: We need to tell that deletion takes time and probably tell that we will delete all Chainstack-added pods, except for cert-manager, installed through helm

**Review by neewy — COMMENTED** [review]
That is a quick review. I'll check the flow for private hosting and will let you know again.

File: docs/platform/deploy-a-consortium-network.md [hunk 9]
```diff
@@ -10,53 +10,63 @@ meta:
...
 
-To deploy a [Hyperledger Fabric](/blockchains/fabric) network:
+## Deploy a Hyperledger Fabric network
 
 1. Select a [consortium project](/glossary/consortium-project) and click **Get started** or **Create network**.
 1. Provide **Network name**.
 1. Under **Blockchain protocol**, select **Hyperledger Fabric**.
 1. The consensus algorithm is [Raft](/blockchains/fabric#consensus). Click **Next**.
-1. Under **Cloud provider**, select your preferred provider and location.
+1. Under **Identity issuer**, select **Cryptogen**.
```
[the remark this answers is not in the document]
Comment (akegaviar) [reply docs/platform/deploy-a-consortium-network.md:19]: It's in a different PR.

File: docs/platform/manage-your-organizations-integrations.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+For instructions on how to set up your Amazon EKS cluster for Chainstack integration, see <a href="https://support.chainstack.com/hc/en-us/articles/900004174426" target="_blank">Setting up an Amazon EKS cluster to integrate with Chainstack</a>.
+
+## Add Amazon EKS as an integration
+
+1. On Chainstack, click <a href="https://console.chainstack.com/integrations" target="_blank">Integrations</a>.
+1. Provide any name.
+1. Under **Type**, select **Amazon Elastic Kubernetes Service**.
+1. Select the region where you have a deployed Amazon EKS service.
+1. Provide the access key and secret of the user you have specifically created for the Chainstack integration.
```
[the remark this answers is not in the document]
Comment (akegaviar) [reply docs/platform/manage-your-organizations-integrations.md:28]: No, it's under `[Manage your organization's integrations](/platform/manage-your-organizations-integrations)`

File: docs/platform/manage-your-organizations-integrations.md (hunk 7 above)
Comment (akegaviar) [reply docs/platform/manage-your-organizations-integrations.md:33]: I don't remember it taking too much time, so I wouldn't.

Comment (akegaviar) [reply docs/platform/manage-your-organizations-integrations.md:33]: I'd rather not.

File: docs/platform/manage-your-organizations-integrations.md (hunk 8 above)
Comment (akegaviar) [reply docs/platform/manage-your-organizations-integrations.md:39]: It's relatively fast, so we don't need to complicate the instructions with the time.

File: docs/platform/manage-your-organizations-integrations.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (neewy) [reply docs/platform/manage-your-organizations-integrations.md:28]: It says
> For instructions on how to set up your Amazon EKS cluster for Chainstack integration

But it does not mention that this KB article explains also IAM roles for a user with programmatic access. This would make me confused. I'd suggest mentioning the topics covered in the KB article, which includes setting up an AWS IAM user for Chainstack Integration, Node Groups sizing for supported protocols, and EKS cluster setup.

File: docs/platform/manage-your-organizations-integrations.md (hunk 6 above)
Comment (neewy) [reply docs/platform/manage-your-organizations-integrations.md:30]: Why did you ignore this suggestion?

**Review by neewy — CHANGES_REQUESTED** [review]
What's missing:
- mention that Integrations are available on Business and Enterprise plans
- a stopped node deployed in private hosting will not incur maintenance costs  
- to deploy a consortium that is spanning across different hostings a network creator has to be on enterprise plan 

Please also mention where are the requirements for AWS user with programmatic access that we use during integration wizard.

**Comment by neewy** [comment]
@easeev I think that we haven't discussed this point and it wasn't implemented yet: 

> a stopped node deployed in private hosting will not incur maintenance costs


Let's discuss that and I will tell Ake what to put in a "stop a node" section

**Comment by neewy** [comment]
We also need to mention customer responsibilities for EKS cluster and resource sizing somewhere, i.e. a customer has to make sure their EKS cluster is operational, they set up snapshots for their storage so that they can recover in case of any accident in AWS, and that IAM user and k8s RBAC rights remain unaffected (otherwise their integration becomes unstable). As for resource sizing please refer to Avi's doc, Node Group section.

**Comment by neewy** [comment]
@akegaviar besides IAM user, there are other prerequisites such as Kubernetes version, etc. Maybe we can add a section in our "Integration" section called "Prerequisites"?

File: docs/platform/manage-your-organizations-integrations.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (akegaviar) [reply docs/platform/manage-your-organizations-integrations.md:28]: Okay, added.

**Comment by akegaviar** [comment]
> mention that Integrations are available on Business and Enterprise plans

Done

> a stopped node deployed in private hosting will not incur maintenance costs

Pending your discussion

> to deploy a consortium that is spanning across different hostings a network creator has to be on enterprise plan

Done

> Please also mention where the requirements are for AWS user with programmatic access that we use during integration wizard.

Done

**Comment by akegaviar** [comment]
> We also need to mention customer responsibilities for EKS cluster and resource sizing somewhere, i.e. a customer has to make sure their EKS cluster is operational, they set up snapshots for their storage so that they can recover in case of any accident in AWS, and that IAM user and k8s RBAC rights remain unaffected (otherwise their integration becomes unstable). As for resource sizing please refer to Avi's doc, Node Group section.

Done in the KB article

**Comment by akegaviar** [comment]
> besides IAM user, there are other prerequisites such as Kubernetes version, etc. Maybe we can add a section in our "Integration" section called "Prerequisites"?

I really want to keep this in the KB and not in the docs. It's all in the KB article.

**Comment by neewy** [comment]
> a stopped node deployed in private hosting will not incur maintenance costs

Approved, let's add this to the docs please.

**Comment by akegaviar** [comment]
> Approved, let's add this to the docs please.

Done

File: docs/platform/manage-your-organizations-identities.md [hunk 11]
```diff
@@ -38,22 +38,22 @@ You may want to export your existing identity from the vault to host a Corda nod
 
 To export an identity:
 
-1. Navigate to [your vault](https://console.chainstack.com/vault) and select the identity to export.
+1. Navigate to <a href="https://console.chainstack.com/vault" target="_blank">your vault</a> and select the identity to export.
```
Comment (neewy) [inline docs/platform/manage-your-organizations-identities.md:41]: why href links?

**Review by neewy — APPROVED** [review]

File: docs/platform/deploy-a-consortium-network.md [hunk 12]
```diff
@@ -10,53 +10,65 @@ meta:
...
 
 1. Select a [consortium project](/glossary/consortium-project) and click **Get started** or **Create network**.
 1. Provide **Network name**.
 1. Under **Blockchain protocol**, select **Hyperledger Fabric**.
 1. The consensus algorithm is [Raft](/blockchains/fabric#consensus). Click **Next**.
-1. Under **Cloud provider**, select your preferred provider and location.
+1. Under **Identity issuer**, select or add an identity issuer. See [Manage your organization's identities](/platform/manage-your-organizations-identities).
+1. Under **Hosting**, select **Chainstack** or **Private**. See [Support hosting options](/platform/supported-hosting-options).
+	* For Chainstack hosting, select a cloud provider.
+	* For private hosting, select an integration. See [Manage your organization's integrations](/platform/manage-your-organizations-integrations).
```
Comment (CaJIbHuK) [inline docs/platform/deploy-a-consortium-network.md:24]:
```suggestion
	* For private hosting, select or add an integration. See [Manage your organization's integrations](/platform/manage-your-organizations-integrations).
```

File: docs/platform/join-a-public-network.md [hunk 13]
```diff
@@ -10,35 +10,45 @@ meta:
...
 
 1. Select a [public chain project](/glossary/public-chain-project) and click **Get started** or **Join network**.
 1. Under **Blockchain protocol**, select **Ethereum**.
 1. Under **Blockchain network**, select **Mainnet**, **Ropsten testnet**, or **Rinkeby testnet**. Click **Next**.
 1. Under **Type**, select whether to run a [shared](/glossary/shared-node) or a [dedicated](/glossary/dedicated-node) node.
 1. Under **Mode**, select whether to run a full node or an archive node. See [Modes](/operations/ethereum/modes).
-1. Under **Cloud provider**, select your preferred provider and location.
+1. Under **Hosting**, select **Chainstack** or **Private**. See [Support hosting options](/platform/supported-hosting-options).
+	* For Chainstack hosting, select a cloud provider.
+	* For private hosting, select an integration. See [Manage your organization's integrations](/platform/manage-your-organizations-integrations).
```
Comment (CaJIbHuK) [inline docs/platform/join-a-public-network.md:22]:
```suggestion
	* For private hosting, select or add an integration. See [Manage your organization's integrations](/platform/manage-your-organizations-integrations).
```

**Review by CaJIbHuK — CHANGES_REQUESTED** [review]

File: docs/platform/manage-your-organizations-identities.md (hunk 11 above)
Comment (akegaviar) [reply docs/platform/manage-your-organizations-identities.md:41]: href links are for chainstack-resources

File: docs/platform/manage-your-organizations-integrations.md [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+  - name: keywords
+    content: private amazon aws cluster integration blockchain deploy kubernetes k8s
+---
+
+# Manage your organization's integrations
+
+You can add the following Kubernetes clusters as an integration to deploy your nodes and networks to:
+
+* [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/)
+* [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/services/kubernetes-service/)
```
[the remark this answers is not in the document]
Comment (akegaviar) [reply docs/platform/manage-your-organizations-integrations.md:14]: @easeev ?

**Review by neewy — DISMISSED** [review]

**Review by CaJIbHuK — APPROVED** [review]

File: docs/platform/manage-your-organizations-integrations.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (easeev) [reply docs/platform/manage-your-organizations-integrations.md:14]: I think the current list is good enough for now

File: docs/.vuepress/config.js [hunk 15]
```diff
@@ -53,8 +54,9 @@ module.exports = {
                     '/platform/delete-a-node',
                     '/platform/delete-a-network',
                     '/platform/delete-a-project',
-                    '/platform/manage-your-organization-identity',
-                    '/platform/view-your-organization-vault',
+                    '/platform/manage-your-organizations-identities',
+                    '/platform/manage-your-organizations-integrations',
+                    '/platform/view-your-organizations-vault',
```
Comment (easeev) [inline docs/.vuepress/config.js:59]: Does `organizations-xxx` sound a bit odd clunky to me only?

Isn't this more consistent?
- Organization identities
- Organization integrations
- Organization vault
- User settings

File: docs/platform/deploy-a-consortium-network.md [hunk 16]
```diff
@@ -10,53 +10,65 @@ meta:
 
 Chainstack supports [cloud](/glossary/cloud) and [hybrid](/glossary/hybrid) deployment of consortium blockchain networks.
 
-To deploy a [Hyperledger Fabric](/blockchains/fabric) network:
+Deployment of multicloud and hybrid hosting networks is available starting from the Enterprise <a href="https://chainstack.com/pricing/" target="_blank">subscription plan</a>.
```
Comment (easeev) [inline docs/platform/deploy-a-consortium-network.md:13]: We should be consistent with terminology on the website, in the platform, in the docs

```suggestion
Deployment of cross-cloud, cross-region and hybrid networks is available starting from the Enterprise <a href="https://chainstack.com/pricing/" target="_blank">subscription plan</a>.
```

File: docs/platform/deploy-a-consortium-network.md [hunk 17]
```diff
@@ -10,53 +10,65 @@ meta:
...
+## Deploy a Hyperledger Fabric network
 
 1. Select a [consortium project](/glossary/consortium-project) and click **Get started** or **Create network**.
 1. Provide **Network name**.
 1. Under **Blockchain protocol**, select **Hyperledger Fabric**.
 1. The consensus algorithm is [Raft](/blockchains/fabric#consensus). Click **Next**.
-1. Under **Cloud provider**, select your preferred provider and location.
+1. Under **Identity issuer**, select or add an identity issuer. See [Manage your organization's identities](/platform/manage-your-organizations-identities).
+1. Under **Hosting**, select **Chainstack** or **Private**. See [Support hosting options](/platform/supported-hosting-options).
+	* For Chainstack hosting, select a cloud provider.
```
Comment (easeev) [inline docs/platform/deploy-a-consortium-network.md:23]: Here and below IMO should be

```suggestion
	* For Chainstack hosting, select a cloud provider and a region.
```

File: docs/platform/stop-or-start-a-node.md [hunk 18]
```diff
@@ -10,7 +10,9 @@ meta:
 
 You can stop and start your deployed nodes.
 
-A stopped node retains its states and does not incur metered compute costs. A stopped node does incur storage costs.
+A stopped node in Chainstack-provided hosting retains its states and does not incur metered compute costs but does incur storage costs.
```
Comment (easeev) [inline docs/platform/stop-or-start-a-node.md:13]: Why "states" not "state"?

File: docs/.vuepress/config.js (hunk 15 above)
Comment (easeev) [reply docs/.vuepress/config.js:59]: No strong opinion here, up to @akegaviar to decide

**Review by easeev — CHANGES_REQUESTED** [review]
Minor changes required

File: docs/.vuepress/config.js (hunk 15 above)
Comment (akegaviar) [reply docs/.vuepress/config.js]: Fixed

File: docs/platform/stop-or-start-a-node.md (hunk 18 above)
Comment (akegaviar) [reply docs/platform/stop-or-start-a-node.md]: Fixed

File: docs/platform/create-a-project.md [hunk 19]
```diff
@@ -8,14 +8,14 @@ meta:
...
-To create a [consortium project](/glossary/consortium-project):
+## Create a consortium project
 
 1. Click **Create project**.
 1. Click **Consortium**.
 1. Provide **Project name** and optionally **Description**.
 1. Click **Create**.
 
-To create a [public chain project](/glossary/public-chain-project):
+## Public chain project
```
Comment (easeev) [inline docs/platform/create-a-project.md:18]: Why "Public chain project" whereas the header above is "Create a consortium project"?

Comment (akegaviar) [reply docs/platform/create-a-project.md:18]: There are two sections. One for consortium, one for public chain.

The word `Create` is missing, hence the confuson.

Fixed.

**Review by easeev — APPROVED** [review]
LGTM
